### DISCUSSION TITLE
ANy plans for Win 11 V22H2 (build 22621.2134) + KB5029263

### DISCUSSION THREAD

**Discussion by SiteSplat** [valinet/ExplorerPatcher#2042] [General]
Any plans of supporting:

Version 22H2 (build 22621.2134) 
with the infamous KB5029263 update? 

@valinet 
🙏 
Thanks!

**Comment by Poopooracoocoo** [comment]
No need to ping the maintainer!

**Comment by SiteSplat** [comment]
is he not the owner of the repo and app?

**Reply by Poopooracoocoo** [reply]
It's just that he's been pinged a lot already about this

**Reply by SiteSplat** [reply]
right, so it's about time to answer once and for all to avoid tons of posts like "crashing" etc etc that just create confusion. Address the issue if you are planning to support it or not with some sort of approx timeframe. Everybody knows life gets in the middle... Would have been so much easier to have an announcement for the problematic KB rather than keep pushing back with "hey that is a preview release, not supported blah blah" (preview release will become a stable release at some point in the update cycle as that is the purpose of the preview release) - seems crazy and unresponsible to tell people to stop win updates. Also no one in the right mind will sponsor the project if this is the way is managed.

**Reply by pyrates999** [reply]
No.  You can just wait like everyone else is.

**Reply by pkaraalphalayer** [reply]
Just for the record, i'm not on any "insider" build and delayed updates for 5 weeks.  I just received [KB5029263], installed it, and it broke my task bar.  I'm uninstalling ExplorerPatcher for now and will return when the issue is resolved.

**Comment by SiteSplat** [comment]
I'm asking for some clarification from the owner of repo... you on the other hand, could stop the machine gun style spam posting around here like you own town. No one, literally no one needs you to go on about pre release blah blah. If you wanted to help (if you are maybe in touch with the owner considering your activity here? ), you could have opened a discussion post about the planned support etc.

**Reply by tallman69** [reply]
I, on the other hand, appreciate the time that pyrates999 puts into helping users that have either inadvertently or purposefully installed a preview or insider build of Windows 11.  Keep up the good work, pyrates999!

**Reply by pyrates999** [reply]
It's nice to be appreciated :)

**Reply by bubbismith** [reply]
I agree with two things here. 

1. @pyrates999 has been quite helpful.
2. It would be nice if we just heard 'someone' was looking into this, even if it's no timeframe.

**Comment by vippium** [comment]
Let's wait for the 23H2 stable release... Will see if valinet is able to manage the compatibility of EP with 23H2. Waiting for next month..
